src/boot_O2/boot_80086760.c

```diff
@@ -1,27 +1,211 @@
+/**
+ * MathF library
+ * Contains tangent function, wrappers for a number of the handwritten functions in fp, and a suite of arctangents
+ */
 #include "global.h"
+#include "fp.h"
```

Comment:
What does `fp` stand for? Why is it called `fp.h`?

Reply:
`fp` is an apparently handwritten file (according to glank, anyway) that implements a load of truncation and rounding operations (indeed, you can see all the functions with helpful names in the header). The header's needed to provide function prototypes, as well as extern the qNaN data. `fp` seems to stand for "fixed point" in this case.

Reply:
Interesting, then I guess my preference would be to call it `fixedPoint.h` instead, same for this file from OoT

Reply:
+1 for calling it fixed point. fp is ambiguous with floating point.